Jenkinsfile

```diff
@@ -6,7 +6,12 @@ node {
   govuk.buildProject(
     rubyLintDiff: false,
     beforeTest: {
-      sh("npm install")
+      stage("Install npm dependencies") {
+        sh("npm install")
+      }
+      stage("Lint Javascript and SCSS") {
+        sh("npm run lint")
```

Comment:
It looks like @tijmenb changed the linting approach yesterday on govuk-jenkinslib [yesterday](https://github.com/alphagov/govuk-jenkinslib/pull/46), so this will end up slightly weird where we have ruby and css linting reported as separate things but JS linting as part of the main build (it's an oddity we'll have on Content Publisher too). So might want to change this to run in a similar way otherwise JS linting will be set-up differently to CSS and Ruby. Might be that there is value in setting up a function in govuk-jenkinslib for linting so there's less boilerplate - I'll drop a comment on Tijmen's PR.

Reply:
Thanks for flagging this @kevindew. I waited for an answer on [your comment](https://github.com/alphagov/govuk-jenkinslib/pull/46#issuecomment-465931313), but since nothing changed unfortunately.

I'm planning to keep the oddity of double linting and merge this PR as is because the longer we wait the harder it is to maintain it as new code comes in without following the linting rules.

Jenkinsfile and tasks can be refactored once an approach is agreed regarding linting.

Let me know if you have major concerns about it.